hollow/src/main/java/com/netflix/hollow/api/codegen/HollowConsumerJavaFileGenerator.java

```diff
@@ -82,4 +136,23 @@ private String createFullPackageName(String packageName, String subPackageName,
     private boolean isEmpty(String value) {
         return value == null || value.trim().isEmpty();
     }
+
+    private void appendImportFromBasePackage(StringBuilder builder, String leaf) {
```

Comment:
Do we need to make sure the firstChar of the leaf is upper case?  

I believe the Code generator upper case the first char of the schema name.

Reply:
The schema name comes in cased correctly in my testing. Also, we can't do it here, since we pass in things like `core.*` and `collections.*` to this method (see :118 and :120)